HFxIISc Collab (#2702)
huggingface/blog: iisc-huggingface-collab.md

---
title: HuggingFace, IISc partner to supercharge model building on India's diverse languages
thumbnail: /assets/iisc-huggingface-collab/thumbnail.png
authors:
- user: prasantgiisc
- user: nihar-artpark
- user: visruth_rf

> - user: Sujith-artpark

Reviewer: Could we have the Hugging Face usernames here please?

Also note, doing this for each entry would create a nice iisc logo under all the names of the author.

```
- user: hf_name
  guest: true
  org: iisc_org_on_huggingface
```

---
# HuggingFace, IISc partner to supercharge model building on India's diverse languages

> The Indian Institute of Science [IISc](https://iisc.ac.in/) and [ARTPARK](https://artpark.in/)  partner with HuggingFace to enable developers across the globe to access [Vaani](https://vaani.iisc.ac.in/), India's most diverse open-source, multi-modal, multi-lingual dataset. Both organisations share a commitment to building inclusive, accessible, and state-of-the-art AI technologies that honor linguistic and cultural diversity.

Reviewer: ```suggestion
The Indian Institute of Science [IISc](https://iisc.ac.in/) and [ARTPARK](https://artpark.in/)  partner with Hugging Face to enable developers across the globe to access [Vaani](https://vaani.iisc.ac.in/), India's most diverse open-source, multi-modal, multi-lingual dataset. Both organisations share a commitment to building inclusive, accessible, and state-of-the-art AI technologies that honor linguistic and cultural diversity.
```

## Partnership

> The partnership between HuggingFace and IISc/ARTPARK aims to increase the accessibility and improve usability of the Vaani dataset, encouraging the development of AI systems that better understand India's diverse languages and cater to the digital needs of its people.

Reviewer: ```suggestion
The partnership between Hugging Face and IISc/ARTPARK aims to increase the accessibility and improve usability of the Vaani dataset, encouraging the development of AI systems that better understand India's diverse languages and cater to the digital needs of its people.
```

## About Vaani Dataset

Launched in 2022 by IISc/ARTPARK and Google, Project Vaani is a pioneering initiative aimed at creating an open-source multi-modal dataset that truly represents India's linguistic diversity. This dataset is unique in its geo-centric approach, allowing for the collection of dialects and languages spoken in remote regions rather than focusing solely on mainstream languages.

Vaani targets the collection of over 150,000 hours of speech and 15,000 hours of transcribed text data from 1 million people across all 773 districts, ensuring diversity in language, dialects, and demographics.

The dataset is being built in phases, with Phase 1 covering 80 districts, which has already been open-sourced. Phase 2 is currently underway, expanding the dataset to 100 more districts, further strengthening Vaani's reach and impact across India's diverse linguistic landscape.

> ![Key Highlights](https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog/iisc-huggingface-collab/Vaani_Dataset_summary.png)

Reviewer: ```suggestion
![Key Highlights](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/iisc-huggingface-collab/Vaani_Dataset_summary.png)
```

* Language Identification Models: Enables the creation of language identification models for various real-world applications.
* Speech Enhancement Systems: The dataset's tagging system supports the development of advanced speech enhancement technologies.
* Enhancing Multimodal LLMs: The unique data collection approach makes it valuable for building and improving multimodal capabilities in LLMs when combined with other multimodal datasets.
* Performance Benchmarking: The dataset is an ideal choice for benchmarking speech models due to its diverse linguistic, geographical, and real-world data properties.

These AI models can power a wide range of Conversational AI applications. From educational tools to telemedicine platforms, healthcare solutions, voter helplines, media localization, and multilingual smart devices, the Vaani dataset can be a game-changer in real-world scenarios.

## What's next

IISc/ARTPARK and Google have extended the partnership to Phase 2 (additional 100 districts). With this, Vaani covers all states in India! We are excited to bring this dataset to all of you.  

> ![Map of districts where data has been collected](https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog/iisc-huggingface-collab/district_map.png)

Reviewer: ```suggestion
![Map of districts where data has been collected](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/iisc-huggingface-collab/district_map.png)
```

*The map highlights the districts across India where data has been collected as of Feb 5,2025*

## How You Can Contribute

The most meaningful contribution you can make is to use the Vaani dataset. Whether building new AI applications, conducting research, or exploring innovative use cases, your engagement helps improve and expand the project.

We would be delighted to hear from you if you have feedback or insights from using the dataset. Please reach out to vaanicontact@gmail.com to share your experiences/inquire about collaboration opportunities or please do fill out this [feedback form](https://docs.google.com/forms/d/e/1FAIpQLSdJ_oMoafkVabj0vgfbTsmECyFFQmbVy3b18NOsxUhYVJKeDQ/viewform).

> ---

Reviewer: Is this an intentional line break?

---
title: HuggingFace, IISc partner to supercharge model building on India's diverse languages
thumbnail: /assets/iisc-huggingface-collab/thumbnail.png
authors:
- user: prasantg
  guest: true
  org: ARTPARK
- user: nihar-artpark
  guest: true
  org: ARTPARK
- user: Visruth-sanka
  guest: true
  org: ARTPARK
- user: SujithPulikodan
  guest: true

>   org: ARTPARK

Reviewer 2: should this be: ARTPARK-IISc?

>   org: iisc_org_on_huggingface

Reviewer: I don't see any org named `iisc_org_on_huggingface`.

Am I missing something?

Author: based on previous suggestion I added this, I though such org is already there. I assume the org we mention here should have an account in HF?

Author: this is resolved

Reviewer: Apologies for not communicating well enough. The idea here is to have an organization on Hugging Face. I see that most of the authors are part of a `ARTPARK-IISc` org. If you add this org name under the author, it would then signify that the author is from the org mentioned (with a logo)

Unfortunately we do not see any org by the name `iisc_org_on_huggingface`.

Please let me know if there are other queries.

## About Vaani Dataset

Launched in 2022 by IISc/ARTPARK and Google, Project Vaani is a pioneering initiative aimed at creating an open-source multi-modal dataset that truly represents India's linguistic diversity. This dataset is unique in its geo-centric approach, allowing for the collection of dialects and languages spoken in remote regions rather than focusing solely on mainstream languages.

Vaani targets the collection of over 150,000 hours of speech and 15,000 hours of transcribed text data from 1 million people across all 773 districts, ensuring diversity in language, dialects, and demographics.

The dataset is being built in phases, with Phase 1 covering 80 districts, which has already been open-sourced. Phase 2 is currently underway, expanding the dataset to 100 more districts, further strengthening Vaani's reach and impact across India's diverse linguistic landscape.

![Key Highlights](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/iisc-huggingface-collab/Vaani_Dataset_summary.png)
*Key Highlights of the Vaani data set, open sourced so far: (as of 15-02-2025)*

> ### District wise language distribution

Reviewer 2: ```suggestion

### District wise language distribution
```
